apps/meteor/app/livechat/server/api/v1/config.ts

```diff
@@ -18,17 +19,17 @@ API.v1.addRoute(
 				return API.v1.success({ config: { enabled: false } });
 			}
 
-			const { token, department, businessUnit } = this.queryParams;
+			const { token, department, businessUnit } = this.queryParams as { token?: string; department?: string; businessUnit?: string };
 
 			const config = await cachedSettings({ businessUnit });
 
 			const status = Livechat.online(department);
-			const guest = token && (await Livechat.findGuest(token));
+			const guest: ILivechatVisitor | null = token ? await Livechat.findGuest(token) : null;
 
-			const room = guest && findOpenRoom(token);
+			const room: IOmnichannelRoom | undefined = guest && token ? findOpenRoom(token) : undefined;
```

Comment:
If there's a guest, there's a token so this feels a bit redundant

Reply:
I know, but I had to add token there to make typescript happy!!

Reply:
I'm welcome to any suggestions about making typescript infer that its not null at this point :)

Reply:
maybe using `guest.token` instead of `token`? 👀